package-lock.json

```diff
@@ -1,84 +1,134 @@
 {
```

Comment:
Hi, @IgorFromUA, please revert changes of this file. In this task you should only add your Readme, not change the settings of whole repo.